contrib/local-environment/docker-compose.yaml

```diff
@@ -47,6 +47,8 @@ services:
   httpbin:
     container_name: httpbin
     image: kennethreitz/httpbin
+    ports:
+      - 8080:80/tcp
```

Comment:
What is the motivation behind exposing httpbin?

Reply:
I want to have a real backend with my locally running and compiled copy of oauth2-proxy